virtcontainers/qemu.go

```diff
@@ -364,6 +364,13 @@ func (q *qemu) createSandbox(sandboxConfig SandboxConfig) error {
 		},
 	}
 
+	// Add bridges before any other devices. This way we make sure that
+	// bridge gets the first available PCI adress i.e bridgePCIStartAddr
+	devices = q.arch.appendBridges(devices, q.state.Bridges)
+	if err != nil {
```

Comment:
Huh? This check looks bogus as `err` wasn't set by `appendBridges()`.

Reply:
yeah you are right, this was part of a later change I am working on.